server/build/webpack.js

```diff
@@ -152,7 +152,7 @@ export default async function createCompiler (dir, { hotReload = false } = {}) {
       root: [
         nodeModulesDir,
         join(dir, 'node_modules')
-      ]
+      ].concat((process.env.NODE_PATH || '').split(';'))
```

Comment:
May be we need to use a split separator like this: 

```
process.platform === 'win32' ? ';' : ':'
```

This is to support windows. CRA and Storybook also follows this pattern.